### ISSUE TITLE
Default values

### ISSUE THREAD

**Issue by rdp** [ivmai/bdwgc#387]
Would it be possible to add the default values to the documentation here? https://github.com/ivmai/bdwgc/blob/master/doc/README.environment That might be nice for knowing which values are typical, thanks!

**Comment by ivmai** [comment]
Sure. If you could propose a patch it would be great!

**Comment by ivmai** [comment]
Same applies to README.macros.

**Comment by ivmai** [comment]
BTW. Out of scope of this issue, but further it would be nice to convert README.environment and README.macros to markdown format (e.g. environment.md and macros.md)

**Comment by rdp** [comment]
Out of scope, might be nice to move the faq into the codebase so it's more easily modifiable :)
Ref: https://github.com/ivmai/bdwgc/blob/master/doc/overview.md

**Comment by ivmai** [comment]
which FAQ do you mean? Overview.md? How to move it closer to the codebase?

**Comment by rdp** [comment]
https://www.hboehm.info/gc/faq.html I don't think it's in the codebase? Thanks.

**Comment by ivmai** [comment]
> https://www.hboehm.info/gc/faq.html I don't think it's in the codebase? Thanks.

Hmm. I can't open it now. Do you have access?

**Comment by ivmai** [comment]
Related: #515, #517

**Comment by rdp** [comment]
Loads OK here?

**Comment by ivmai** [comment]
Yes, I have access to faq.html now. I am OK if you put it to doc folder but let's convert it to .md (in a separate commit).

**Comment by ivmai** [comment]
> https://www.hboehm.info/gc/faq.html I don't think it's in the codebase? Thanks.

I've put it to the codebase (https://github.com/ivmai/bdwgc/commit/ba42b3585b53e460f944d64e39edc086ff13daaf) and converted to .md file ([doc/faq.md](https://github.com/ivmai/bdwgc/blob/master/doc/faq.md)).
